R/filters.R

```diff
@@ -117,3 +119,31 @@ filter_by_repertoire <- function(.data, .min_size) {
 filter_by_clonotype <- function(.data, .query) {
   return(list(data = .data$data, meta = .data$meta))
 }
+
+include <- function(...) {
+  args <- unlist(unname(list(...)))
```

Comment:
why do you use unname everywhere?

Reply:
How to handle the case if user will specify names? Stop with error?